packages/tailwindcss-language-server/src/oxide.ts

```diff
@@ -58,10 +58,32 @@ declare namespace OxideV3 {
   }
 }
 
+// This covers the Oxide API from v4.1.0+
+declare namespace OxideV5 {
+  interface GlobEntry {
+    base: string
+    pattern: string
+    negated: boolean
```

Comment:
```suggestion
```

The `GlobEntry` itself doesn't have a negated flag. Negated globs are not returned to `@tailwindcss/vite`, `@tailwindcss/postcss` or `@tailwindcss/cli`.